pkg/graph/resolvers/plan_task_list_section_locks.go

```diff
@@ -253,7 +253,7 @@ func onLockTaskListSectionUnsubscribeComplete(
 		_, err := UnlockTaskListSection(ps, modelPlanID, section, subscriber.GetPrincipal().Account().ID, model.ActionTypeNormal)
 
 		if err != nil {
-			fmt.Printf("Uncapturable error on websocket disconnect: %v\n", err.Error())
+			fmt.Printf("Uncapturable error on websocket disconnect: %v\n", err.Error()) //TODO: can we pass a reference to the logger to the pubsub?
```

Comment:
This should be provided to pubsub on service creation rather than passed as a dependency in function calls.

Reply:
agreed! I just wanted to add a note for now